server/backend/testspg/router/test_router_application.py

```diff
@@ -419,6 +417,9 @@ def test_get_user_role_assignments_filtering_for_bceid_delegated_admin(
         fom_dev_access_admin_token, access_grants_not_able_to_see
     )
 
+    # override router guard dependencies
+    override_enforce_bceid_terms_conditions_guard()
```

Comment:
Fix broken tests that is for BCeID delegated admin requester; `override_enforce_bceid_terms_conditions_guard` defaults to pass the check.